packages/core/admin/server/src/controllers/authentication.ts

```diff
@@ -11,59 +12,66 @@ import {
   validateRenewTokenInput,
 } from '../validation/authentication';
 
+import type {
+  ForgotPassword,
+  Login,
+  Register,
+  RegistrationInfo,
+  RenewToken,
+  ResetPassword,
+} from '../../../shared/contracts/authentication';
```

Comment:
@joshuaellis and I were discussing at some point if we want to introduce module aliases with the switch to vite which would allow you to do this. I'd be very much in favor of it, because if would make these long import paths more readable.

Reply:
Yeah we could alias so it's like `@shared/contracts` – you can use `paths` in tsconfig to do this, we'd _maybe_ have to add aliasing to the webpack config / jest & maybe pack-up but it does make things better.

Reply:
that would be :cheff-kiss: . For now we will continue with the relative import, and lets iterate later on this